src/vit-based/vit.jl

```diff
@@ -100,9 +102,10 @@ end
 @functor ViT
 
 function ViT(config::Symbol; imsize::Dims{2} = (256, 256), patch_size::Dims{2} = (16, 16),
-             pretrain::Bool = false, inchannels::Integer = 3, nclasses::Integer = 1000)
+             pretrain::Bool = false, inchannels::Integer = 3, nclasses::Integer = 1000, 
+             qkv_bias=false)
```

Comment:
Unless it is typical to adjust this toggle, I think it should not get exposed going from `vit` to `ViT`. The logic with the codebase has been to make the uppercase exports as simple as possible.

Reply:
I add to add it since the default for torchvision is `true`, here was `false`. The torchvision model is given by
```

Reply:
Okay, so change the default to `true` and remove the keyword? I assume you almost always want it as `true`.

Reply:
yes, I'll do it in the next PR